src/common/commonutils/UserUtils.c

```
@@ -395,14 +395,14 @@ int EnumerateUsers(SIMPLIFIED_USER** userList, unsigned int* size, char** reason
     }
     else
     {
-        OsConfigLogError(log, "EnumerateUsers: cannot read %s", passwdFile);
+        OsConfigLogInfo(log, "EnumerateUsers: cannot read %s", passwdFile);
```

I think this might be out of scope but the this can fail if GetNumberOfLinesInFile fails and function \can fail if GetNumberOfCharacterInstancesInFile which hides errors from fopen and fclose where error condition is not checked.  So ideally we could improve the error handling there, add more tests and then change this line to log error if system error is found if so then it probably makes sense in separate PR

---

GetNumberOfLinesInFile only fails if the file does not exist or if it cannot be open in read mode (same thing). We do not need to alarm users about this, as serious as it may sound, it's not a fatal error for us and we can complete the audit. So keeping this one also as [INFO].